src/Components/Articles/Articles.jsx

```diff
@@ -0,0 +1,75 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../../AxiosApi/axiosApi";
+import "./Articles.modules.css";
+import sortOptions from "../../utils/__tests__/sortOptions";
+import ArticleCard from "./ArticleCard";
+import PageSetter from "./PageSetter";
+import { useLocation, useNavigate } from "react-router-dom";
+
+function Articles() {
+  const navigate = useNavigate()
+  const location = useLocation();
+  const query = new URLSearchParams(location.search);
+
+  const [articlesData, setarticlesData] = useState({
+    articles: [],
+    articlesCount: "getting articles",
+  });
+  const [sortedBy, setSortedBy] = useState({
```

Comment:
Although this will be useful later on, there's no requirement to sort the articles at this stage, so it shouldn't be included in this ticket